deno_dist/utils/stream.ts

```diff
@@ -0,0 +1,39 @@
+export class StreamingApi {
+  private writer: WritableStreamDefaultWriter<Uint8Array>
+  private encoder: TextEncoder
+  private writable: WritableStream
+
+  constructor(writable: WritableStream) {
+    this.writable = writable
+    this.writer = writable.getWriter()
+    this.encoder = new TextEncoder()
+  }
+
+  async write(input: Uint8Array | string) {
+    if (typeof input === 'string') {
+      input = this.encoder.encode(input)
+    }
+    await this.writer.write(input)
+    return this
+  }
+
+  async writeln(input: string) {
+    await this.write(input + '\n')
+    return this
+  }
+
+  async log(arg: any) {
```

Comment:
I've thought it.`log()` is good, but to reduce bundle size, I think we shouldn't add this. It's a so slight change, but we're nervous about increasing bundle sizes.

OK @geelen ?

Reply:
@sor4chi 

Let's remove the `log()`!